tests/general/cloc_test_files/example2.py

```diff
@@ -0,0 +1,7 @@
+# This comments are for testing purpose
+# This program will just try to add two static numbers
+
+first = 5  # First number
+second = 10  # Second number
+
+sum = first + second
```

Comment:
It is an existing bug, btw your editor seems to be crlf I think, if you add newline at the end of the file, this shouldn't show up I think..